### ISSUE TITLE
MissingPluginException(No implementation found for method handleCardAction on channel flutter.stripe/payments)

### ISSUE THREAD

**Issue by Flajt** [flutter-stripe/flutter_stripe#743]
**Describe the bug**
Every time I use a Card with 3D Secure, the package throws this error.
The setup intent works fine, no issues there, 3D Secure is handled as it should.

EDIT: I found that there is a mention in release 2.5.0 that you depreciated `handleCardAction` in favor of `handleNextAction`, but the latter isn't defined, maybe this has something to do with it?

**To Reproduce**
1. Create a setupIntent for storing payment information for later uses
2. Use the following test card for the setup intent: `4000 0027 6000 3184`
3. Verify 3D Secure successful
4. Create a payment intent with the card
5. Watch it throwing an error

**Expected behavior**
The 3D Secure redirection should happen

**Smartphone / tablet**
 - Device: A51
 - OS: Android 13
 - Package version: 3.0.0
 - Flutter version: 3.0.1
 - distributionUrl: https\://services.gradle.org/distributions/gradle-7.4-all.zip
 - kotlin_version = '1.6.21'  (the new one)
 - kotlin_version = '1.5.10' (old one)
 - grandle_version = '7.1.2' (new one)
 - grandle_version = '4.2.0' (old one)
 - mind_sdk_version: 21

**Additional context**
- I've run `flutter pub cache repair` 2x already
- I've run `flutter pub clean` 3x times
- I've also tried restarting my laptop, to no avail
- I've downgraded my grandle version and kotlin version, but it doesn't seem to be the issue (I've updated them to match the version in the docs a 2 days ago)
- I thought reinstalling the app on my device should fix it, but apparently it doesn't

**Code:**
My pay method:
```dart

Future<void> pay(PaymentBlueprint blueprint) async {
    if (blueprint.customerSecret != "") {
      Stripe.stripeAccountId = blueprint.merchantId;

      await Stripe.instance.initPaymentSheet(
          paymentSheetParameters: SetupPaymentSheetParameters(
              googlePay: Platform.isAndroid
                  ? true
                  : false, //await Stripe.instance.isGooglePaySupported(const IsGooglePaySupportedParams()),
              applePay: Stripe.instance.isApplePaySupported.value,
              customFlow: false,
              customerId: blueprint.customerId,
              customerEphemeralKeySecret: blueprint.customerSecret,
              paymentIntentClientSecret: blueprint.clientKey,
              style: ThemeMode.light,
              primaryButtonColor: Colors.deepOrange,
              merchantCountryCode: blueprint.countryCode,
              merchantDisplayName: blueprint.merchantName,
              testEnv: true));
      await Stripe.instance.presentPaymentSheet();
    }
  }
```
Setup method:
```dart
Future<void> setupPaymentMethod(String secret, [paymentMethod]) async {
    try {
      await Stripe.instance.initPaymentSheet(
          paymentSheetParameters: SetupPaymentSheetParameters(
              merchantDisplayName: "Simple Checkout",
              allowsDelayedPaymentMethods: true,
              primaryButtonColor: Colors.deepOrange,
              googlePay: Platform.isAndroid
                  ? true
                  : false, //await Stripe.instance.isGooglePaySupported(const IsGooglePaySupportedParams(testEnv: true)) Buggy
              applePay: Stripe.instance.isApplePaySupported.value,
              merchantCountryCode: "DE",
              setupIntentClientSecret: secret,
              currencyCode: "EUR",
              style: ThemeMode.light,
              testEnv: true));

      await Stripe.instance.presentPaymentSheet();
    } catch (e) {
      print(e);
    }
  }
```
3D Secure confirmation call
```dart
Future<bool> confirm(PaymentBlueprint blueprint) async {
    Stripe.stripeAccountId = blueprint.merchantId;
    await Stripe.instance.handleCardAction(blueprint.customerSecret ?? "");
    return true;
  }
```

**Comment by Flajt** [comment]
I've been digging a bit in the android implementation and found out that a change in the changelog for `2.5.0` has been added [here](https://github.com/flutter-stripe/flutter_stripe/blob/main/packages/stripe_android/android/src/main/kotlin/com/flutter/stripe/StripeAndroidPlugin.kt) as a method call: `handleNextAction` in line 105.

However: In [here](https://github.com/flutter-stripe/flutter_stripe/blob/main/packages/stripe/lib/src/stripe.dart) it's still called: `handleCardAction` and no `handleNextAction` can be found.

Also [here](https://github.com/flutter-stripe/flutter_stripe/blob/main/packages/stripe_platform_interface/lib/src/method_channel_stripe.dart) in line: 129, you dispatch a call to `handleCardAction` not to `handleNextAction`.

As I'm not good in native development I might be false, but maybe that is the issue: wrong channel calls and wrongly named methods.

**Comment by jonasbark** [comment]
This was fixed by https://github.com/flutter-stripe/flutter_stripe/pull/733 and released with 3.0.1
